region/brinstar/green.json

```diff
@@ -2291,27 +2339,24 @@
                   "name": "Freeze the Beetoms",
                   "notable": false,
                   "requires": [
-                    "Morph",
                     "Ice"
                   ]
                 },
                 {
                   "name": "Tank the Damage",
                   "notable": false,
                   "requires": [
-                    "Morph",
                     {"enemyDamage": {
                       "enemy": "Beetom",
                       "type": "contact",
-                      "hits": 15
```

Comment:
Maybe needs another strat for spike iframing the beetoms. 3 spike hits to cross, maybe one more for safety.
There's also some fuzziness you can do by shaking off the beetoms and running to a farm point.  But that doesn't seem worthwhile enough to add.